### ISSUE TITLE
Update module settings

### ISSUE THREAD

**Issue by KenitoInc** [microsoftgraph/entra-powershell#1054]
Thanks for reporting the bug. Please ensure you've gone through the following checklist before opening an issue:
- Make sure you can reproduce this issue using the latest released version of `Microsoft.Graph.Entra` or `Microsoft.Graph.Entra.Beta`.
- Please search the existing issues to see if there has been a similar issue filed.

**Describe the bug**
> A clear and concise description of what the bug is.
From the [guidelines](https://learn.microsoft.com/en-us/powershell/scripting/dev-cross-plat/performance/module-authoring-considerations?view=powershell-7.4#guidelines), we should author exported cmdlets as follows
```powershell
@{
    FunctionsToExport = 'Format-Hex', 'Format-Octal'
    CmdletsToExport = @()  # Specify an empty array, not $null
    AliasesToExport = @()  # Also ensure all three entries are present
}
```

However, this is how we have configured our module
<img width="600" alt="image" src="https://github.com/user-attachments/assets/7a4229fc-c348-4c08-8541-132e8eff45de">

**To Reproduce**
1. Execute `-Entra` with `...`
2. See error at `...`

**Expected behavior**
> A clear and concise description of what you expected to happen.

**Debug Output**
> Run the problematic command with `-Debug` and paste the resulting debug stream below.
> ⚠ ATTENTION: Be sure to remove any sensitive information that may be in the logs.

**Module Version**
> Please run `Get-Module Microsoft.Graph.Entra*` after cmdlet execution and paste the output below.
> If a module cannot be installed or imported, please run `Get-Module -ListAvailable` and paste the output.

**Environment Data**
> Please run `$PSVersionTable` and paste the output below. If running the Docker container image, indicate the tag of the image used and the version of Docker engine.

**Screenshots**
> If applicable, add screenshots to help explain your problem.

**Additional context**
> Add any other context about the problem here.

**Comment by SamErde** [comment]
I've seen a few build scripts in comunity repos that can pull the function names from all PS1 files in the build scope and then either add then to the manifest or perform a check to make sure they're all listed as an array in the manifest.

**Comment by KenitoInc** [comment]
Fixed by #1097
